### PR TITLE
url-shortener: helm chart with cue config file

### PR DESCRIPTION
**Pull request by DTabidze**

### CODE REVIEW

File: charts/url-shortener/templates/install.yaml [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+      containers:
+      - name: url-shortener
+        image: {{ .Values.image.repository }}:{{ .Values.image.tag }}
+        ports:
+        - name: http
+          containerPort: {{ .Values.port }}
+          protocol: TCP
+        command: 
+        - url-shortener
+        - --port="{{ .Values.port }}
```
Comment (giolekva) [inline charts/url-shortener/templates/install.yaml:70]: rm `"`

File: core/installer/values-tmpl/url-shortener.cue [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+	urlShortener: {
+		repository: "giolekva"
+		name: "url-shortener"
+		tag: "latest"
+		pullPolicy: "Always"
+	}
+}
+
+charts: {
+    url-shortener: {
```
Comment (giolekva) [inline core/installer/values-tmpl/url-shortener.cue:24]: Can not use `-` in the names, make it `urlShortener`

File: charts/url-shortener/Chart.yaml [hunk 3]
```diff
@@ -0,0 +1,6 @@
+apiVersion: v2
+name: url-shortener
+description: A Helm chart for URL shortener application
+type: application
+version: 0.1.0
+appVersion: "1.0"
```
Comment (giolekva) [inline charts/url-shortener/Chart.yaml:6]: Use `0.0.1` instead

File: core/installer/values-tmpl/url-shortener.cue
Comment (giolekva) [inline core/installer/values-tmpl/url-shortener.cue:1]: App Store to pick up new application you'll have to add path to the cue file here: https://github.com/giolekva/pcloud/blob/83b7219dc678a0e40751caf594f652f73a888266/core/installer/app.go#L35

**Review by giolekva — CHANGES_REQUESTED** [review]

File: core/installer/values-tmpl/url-shortener.cue [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+namespace: "app-url-shortener"
+readme: "URL shortener application will be installed on \(input.network.name) network and be accessible at https://\(_domain)"
+description: "Provides URL shortening service. Can be configured to be reachable only from private network or publicly."
+icon: "<svg xmlns='http://www.w3.org/2000/svg' width='50' height='50' viewBox='0 0 256 256'><path fill='currentColor' d='M100 140a8 8 0 1 1-8-8a8 8 0 0 1 8 8Zm64 8a8 8 0 1 0-8-8a8 8 0 0 0 8 8Zm64.94-9.11a12.12 12.12 0 0 1-5 1.11a11.83 11.83 0 0 1-9.35-4.62l-2.59-3.29V184a36 36 0 0 1-36 36H80a36 36 0 0 1-36-36v-51.91l-2.53 3.27A11.88 11.88 0 0 1 32.1 140a12.08 12.08 0 0 1-5-1.11a11.82 11.82 0 0 1-6.84-13.14l16.42-88a12 12 0 0 1 14.7-9.43h.16L104.58 44h46.84l53.08-15.6h.16a12 12 0 0 1 14.7 9.43l16.42 88a11.81 11.81 0 0 1-6.84 13.06ZM97.25 50.18L49.34 36.1a4.18 4.18 0 0 0-.92-.1a4 4 0 0 0-3.92 3.26l-16.42 88a4 4 0 0 0 7.08 3.22ZM204 121.75L150 52h-44l-54 69.75V184a28 28 0 0 0 28 28h44v-18.34l-14.83-14.83a4 4 0 0 1 5.66-5.66L128 186.34l13.17-13.17a4 4 0 0 1 5.66 5.66L132 193.66V212h44a28 28 0 0 0 28-28Zm23.92 5.48l-16.42-88a4 4 0 0 0-4.84-3.16l-47.91 14.11l62.11 80.28a4 4 0 0 0 7.06-3.23Z'/></svg>"
```
Comment (giolekva) [inline core/installer/values-tmpl/url-shortener.cue:12]: Use this instead: `<svg xmlns="http://www.w3.org/2000/svg" width="40.63" height="50" viewBox="0 0 13 16"><circle cx="2" cy="10" r="1" fill="currentColor"/><circle cx="2" cy="6" r="1" fill="currentColor"/><path fill="currentColor" d="M4.5 14c-.06 0-.11 0-.17-.03a.501.501 0 0 1-.3-.64l4-11a.501.501 0 0 1 .94.34l-4 11c-.07.2-.27.33-.47.33m3 0c-.06 0-.11 0-.17-.03a.501.501 0 0 1-.3-.64l4-11a.501.501 0 0 1 .94.34l-4 11c-.07.2-.27.33-.47.33"/></svg>`

File: core/installer/values-tmpl/url-shortener.cue (hunk 2 above)
Comment (DTabidze) [reply core/installer/values-tmpl/url-shortener.cue:24]: Done

File: core/installer/values-tmpl/url-shortener.cue
Comment (DTabidze) [reply core/installer/values-tmpl/url-shortener.cue:1]: Done

File: core/installer/values-tmpl/url-shortener.cue (hunk 4 above)
Comment (DTabidze) [reply core/installer/values-tmpl/url-shortener.cue]: Done

**Review by giolekva — APPROVED** [review]

**Pull request merged**